README.md

```diff
@@ -1441,6 +1441,21 @@ Other Style Guides
     import barCss from 'bar.css';
     ```
 
+  <a name="modules--import-extensions"></a>
+  - [10.10](#modules--import-extensions) Do not include javascript filename extensions 
```

Comment:
See [the build logs](https://travis-ci.org/airbnb/javascript/jobs/657142404#L304), where it says:
```
README.md:1445 MD009/no-trailing-spaces Trailing spaces [Expected: 0; Actual: 1]
```